cmd/lotus-seed/main.go

```diff
@@ -92,3 +102,185 @@ var preSealCmd = &cli.Command{
 		return seed.WriteGenesisMiner(maddr, sbroot, gm)
 	},
 }
+
+var aggregateManifestsCmd = &cli.Command{
+	Name:  "aggregate-manifests",
+	Usage: "aggregate a set of preseal manifests into a single file",
+	Action: func(cctx *cli.Context) error {
+		var inputs []map[string]genesis.GenesisMiner
+		for _, infi := range cctx.Args().Slice() {
+			fi, err := os.Open(infi)
+			if err != nil {
+				return err
+			}
+			defer fi.Close()
+			var val map[string]genesis.GenesisMiner
+			if err := json.NewDecoder(fi).Decode(&val); err != nil {
+				return err
+			}
+
+			inputs = append(inputs, val)
+		}
+
+		output := make(map[string]genesis.GenesisMiner)
+		for _, in := range inputs {
+			for maddr, val := range in {
+				if gm, ok := output[maddr]; ok {
+					output[maddr] = mergeGenMiners(gm, val)
+				} else {
+					output[maddr] = val
+				}
+			}
+		}
+
+		blob, err := json.MarshalIndent(output, "", "  ")
+		if err != nil {
+			return err
+		}
+
+		fmt.Println(string(blob))
+		return nil
+	},
+}
+
+var aggregateSectorDirsCmd = &cli.Command{
+	Name:  "aggregate-sector-dirs",
+	Usage: "aggregate a set of preseal manifests into a single file",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:  "miner",
+			Usage: "Specify address of miner to aggregate sectorbuilders for",
+		},
+		&cli.StringFlag{
+			Name:  "dest",
+			Usage: "specify directory to create aggregate sector store in",
+		},
+		&cli.Uint64Flag{
+			Name:  "sector-size",
+			Usage: "specify size of sectors to aggregate",
+			Value: 32 * 1024 * 1024 * 1024,
```

Comment:
If you want you can default to the same thing as everything else `pre-seal` uses `build.SectorSizes[0]` for `--sector-size`.